mathjax3-ts/input/tex/verb/VerbConfiguration.ts

```diff
@@ -0,0 +1,70 @@
+/*************************************************************
+ *
+ *  Copyright (c) 2018 The MathJax Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+/**
+ * @fileoverview Configuration file for the verb package.
+ *
+ * @author v.sorge@mathjax.org (Volker Sorge)
+ */
+
+import {Configuration} from '../Configuration.js';
+import {TexConstant} from '../TexConstants.js';
+import TexParser from '../TexParser.js';
+import {CommandMap} from '../SymbolMap.js';
+import {ParseMethod} from '../Types.js';
+import TexError from '../TexError.js';
+
+
+// Namespace
+export let VerbMethods: Record<string, ParseMethod> = {};
+
+
+/**
+ * Implements the verbatim notation \verb|...|.
+ * @param {TexParser} parser The current tex parser.
+ * @param {string} name The name of the calling macro.
```

Comment:
Missing jsdoc for `notation: string` argument.

Reply:
Actually `notation` is never used. Must have been left over from copy/paste from `CancelConfiguration`.